firstmicroservice/src/main/java/com/stgcodes/validation/AddressValidator.java

```diff
@@ -38,23 +39,23 @@ private void validateLineTwo(Address address, Errors errors) {
         String lineTwo = address.getLineTwo().replaceAll(WHITESPACE_DASH_SLASH_MATCHER, "");
 
         if (lineTwo.length() > 25 || lineTwo.length() == 0) {
-            errors.rejectValue("linetwo", "linetwo.format");
+            errors.rejectValue("lineTwo", "linetwo.format");
         }
     }
 
     private void validateCity(Address address, Errors errors) {
         String city = address.getCity().replaceAll(WHITESPACE_DASH_SLASH_MATCHER, "");
 
-        if (city.length() > 50 || city.length() == 0) {
+        if (city.length() > 21 || city.length() == 0) {
```

Comment:
This logic doesn't appear to be correct. I'm not sure we want to restrict the length of city to 21.